Makefile

```diff
@@ -14,27 +15,83 @@ else
 	DOCKERFILE=Dockerfile
 endif
 
-.PHONY: build
+############
+## Docker ##
+############
+
 build: ## Build a Docker image
 	docker build \
 		--pull \
 		--tag freshrss/freshrss:$(TAG) \
-		-f Docker/$(DOCKERFILE) .
+		--file Docker/$(DOCKERFILE) .
 
-.PHONY: start
 start: ## Start the development environment (use Docker)
 	docker run \
 		--rm \
-		-v $(shell pwd):/var/www/FreshRSS:z \
-		-p $(PORT):80 \
-		-e FRESHRSS_ENV=development \
+		--volume $(shell pwd):/var/www/FreshRSS:z \
+		--publish $(PORT):80 \
+		--env FRESHRSS_ENV=development \
 		--name freshrss-dev \
 		freshrss/freshrss:$(TAG)
 
-.PHONY: stop
 stop: ## Stop FreshRSS container if any
 	docker stop freshrss-dev
 
-.PHONY: help
+.PHONY: build start stop
+
+##########
+## I18N ##
+##########
+i18n-format: ## Format I18N files
+	@$(PHP) ./cli/manipulate.translation.php -a format
+	@echo Files formatted.
+
+i18n-add-language: ## Add a new supported language
+ifndef lang
+	@echo To add a new language, you need to provide one in the "lang" variable.
+	@exit 10
+endif
+	@$(PHP) ./cli/manipulate.translation.php -a add -l $(lang)
+	@echo Language added.
+
+i18n-add-key: ## Add a key
+ifndef key
+	@echo To add a key, you need to provide one in the "key" variable.
+	@exit 10
+endif
+ifndef value
+	@echo To add a key, you need to provide its value in the "value" variable.
+	@exit 10
+endif
+	@$(PHP) ./cli/manipulate.translation.php -a add -k $(key) -v $(value)
+	@echo Key added.
+
+i18n-remove-key: ## Remove a key
+ifndef key
+	@echo To remove a key, you need to provide one in the "key" variable.
+	@exit 10
+endif
+	@$(PHP) ./cli/manipulate.translation.php -a delete -k $(key)
+	@echo Key removed.
+
+i18n-ignore-key: ## Ignore a key for the selected language
+ifndef lang
+	@echo To ignore a key, you need to provide a language in the "lang" variable.
+	@exit 10
+endif
+ifndef key
+	@echo To ignore a key, you need to provide one in the "key" variable.
+	@exit 10
+endif
+	@$(PHP) ./cli/manipulate.translation.php -a ignore -k $(key) -l $(lang)
+	@echo Key ignored.
+
+.PHONY: i18n-format i18n-add-language i18n-add-key i18n-remove-key i18n-ignore-key
```

Comment:
I prefer to add `.PHONY` next to the commands because it helps to don't forget them when adding a new target

Reply:
Done